api/specs.proto

```diff
@@ -274,6 +274,9 @@ message ClusterSpec {
 
 	// TaskDefaults specifies the default values to use for task creation.
 	TaskDefaults task_defaults = 7 [(gogoproto.nullable) = false];
+
+	// LockKey is the encryption key for the raft data
```

Comment:
For now, as I understand, this is limited to raft. Do we have plans on opening this up to "at rest" data? Would it be more accurate to say this?